addons/account/__init__.py

```diff
@@ -50,6 +50,8 @@ def _auto_install_l10n(env):
                 module_list.append('l10n_generic_coa')
         if country_code in ['US', 'CA']:
             module_list.append('account_check_printing')
+        if country_code in ['AR']:
+            module_list.append('l10n_latam_check')
```

Comment:
Hi @william-andre 
Thanks!
I didn't know about the auto_install parameter with the dependencies, that's a nice one :)
IMHO:
1. is not really mandatory to auto install this module, but as we remove the "application=True" and it's not display on any config, auto-installing could be a good option for usability and most Argentiniean companies would need it.

2. I would not make l10n_ar depend on this one because it's not a real dependency and I belieive keeping optional (at lease technically) is better

3. setting on this module  'auto_install': True or 'auto_install': ['l10n_ar', 'account_check_printing'] could work but the problem is that this module would be installed after l10n_ar and after the chart being installed. And this module is inheriting chart installation methods to create additional journals.

As it's now, on a fresh database, if the main company is Argentina and you install accounting, this module is installed first and then l10n_ar, and then the chart is installed. So everything runs smoothly. And this behaviour is the one when you create a database on odoo online. 

I would keep as it is, but if for some reason is not a good idea maybe we van remove for now (it's not a big deal)

cc @jco-odoo @vbe-odoo

Reply:
as @jco-odoo request, removed all this changes on the account hook